docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -76,61 +79,55 @@ type. Example on how to use config options was presented below:
     # assigning text string "some_text" value to the option
     CONFIG_SAMPLE_STRING_OPTION="some_text"
 
-For more detailed information visit
-[Setting Kconfig values](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/setting.html#setting-configuration-values).
+For more detailed information, read about [setting Kconfig values](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/setting.html#setting-configuration-values)
+in the nRF Connect SDK documentation.
 
-Please keep in mind that Kconfig files are used in building process, so after
-changing them you have rebuild your application (see:
-[Building nRF Connect examples](TODO:) for more details), by typing following
-command in the example directory:
+Because Kconfig files are used in the building process, make sure that you rebuild your application
+after editing them by typing the following command in the example directory:
 
-        # <board_name> should be replaced with the actual target board name
-        $ west build -b <board_name>
+        # board_name should be replaced with the actual target board name
+        $ west build -b board_name
+
+See Building nRF Connect examples for details.
```

Comment:
I guess there won't be such a guide, building will be described in the examples' READMEs